spec/app_spec.rb

```diff
@@ -0,0 +1,5 @@
+require "app"
+
+RSpec.describe "rock, paper, scissors" do
+  it "has a route to a start page" do
+    expect(page).to \
```

Comment:
I figured out the feature test structure from this page in the pairing challenges, hopefully that helps it make sense. https://github.com/makersacademy/course/blob/main/apprenticeships_intro_to_the_web/walkthroughs/entering_players.md